mobile/lib/widgets/asset_grid/multiselect_grid.dart

```diff
@@ -204,10 +204,11 @@ class MultiselectGrid extends HookConsumerWidget {
       processing.value = true;
       try {
         final localIds = selection.value.where((a) => a.isLocal).toList();
```

Comment:
The best way to handle this is to use the `onlyBackedUp` flag to filter assets directly here instead of having the filtering logic in the provider. Having the logic here will also allow you to show the toast when there are no assets to delete.